docs/dev/connectors/filesystem_sink.md

```diff
@@ -23,6 +23,10 @@ specific language governing permissions and limitations
 under the License.
 -->
 
+<div class="alert alert-info">
+The BucketingSink has been deprecated since Flink 1.9 and will be removed in subsequent releases. Please use the StreamingFileSink instead.
```

Comment:
let'S add a link to the documentation of the StreamingFileSink

Reply:
I tried to do so but it is not trivial to put a link in a banner like this. I will try.

Reply:
Done.